WalletWasabi.Gui/Controls/StatusBar.xaml

```diff
@@ -84,6 +86,8 @@
         <Panel Height="10" Width="10" IsVisible="{Binding CriticalUpdateAvailable}">
           <DrawingPresenter VerticalAlignment="Center" HorizontalAlignment="Center" Drawing="{DynamicResource StatusBar_YellowFlag}" />
         </Panel>
+        <TextBlock Text="{Binding Network, Converter={StaticResource NetworkStringConverter}}" />
```

Comment:
From the beginning, Wasabi is not showing that you are on the MainNet to avoid confusing users who are not familiar with Networks. Now we have this: 

![image](https://user-images.githubusercontent.com/9844978/89911497-cc55d580-dbf1-11ea-83ac-033c9ac162ec.png)

This is changing this concept - I am against this.

Reply:
Ok, makes sense, i am going to remove it.